scripts/test_blip_caption_cleanup.py

```
@@ -0,0 +1,154 @@
+
+def run_cpu_test(model_module):
+    # Track gc.collect calls by monkeypatching the module's gc.collect
+    gc_calls = [0]
+    orig_gc_collect = model_module.gc.collect
+    model_module.gc.collect = lambda: gc_calls.__setitem__(0, gc_calls[0] + 1)
+
+    # Build model instance with stub methods
+    m = model_module.TritonPythonModel()
+    m._device = "cpu"
+    # No heavy torch usage in CPU path for our stubs
+    m._decode_image = lambda b64: CloseCounterImage(close_counter)
```

Ruff (0.14.2)

Unused lambda argument: `b64`

(ARG005)